access_point/server/server.py

```diff
@@ -16,28 +16,35 @@ def index():
 def install():
     return 'starting the installation script'
 
-@app.route('/config/<stt>/<tts>/<hotword>/<wake>')
-def config(stt, tts, hotword, wake):
-    subprocess.call(['sudo', 'bash' , access_point_folder + '/config.sh', stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
-    display_message = {"Configuration":"Successful"}
+@app.route('/config',methods=['GET'])
+def config():
+    stt = request.args.get('stt')
+    tts = request.args.get('tts')
+    hotword = request.args.get('hotword')
+    wake = request.args.get('wake')
+    subprocess.call(['sudo bash ' + access_point_folder + '/config.sh ' + ' {} {} {} {}  & '.format(stt, tts, hotword, wake)],shell=True)  #nosec #pylint-disable type: ignore
```

Comment:
What?? Old mistake repeated?

Reply:
@hongquan , what mistake??

Reply:
if you are referring to shell=True , we need to use '&' operator , hence we are using shell=True

Reply:
Even with `shell=True`, your call of `subprocess` is already wrong.
But, using shell is not encouraged. #254

Reply:
But we need to use shell operator in this

Reply:
@hongquan , please explain what do you mean by its "wrong" ? 🤔  ?  It works fine on my side.

Reply:
Just read the doc. Read carefully.

Working fine at this moment doesn't mean it is always working. To satisfy too early about "it is working" is the reason why you used to produce a lot of bugs.

Again, didn't I clearly require:

> Check again if we really need the feature of shell & operator.
>  If it is needed, implement that feature in Python (without using shell).
?

Reply:
Yes @hongquan . Its needed , else the pi will restart before sending response and would lead the app to crash even if the Speaker is configured successfully

Reply:
The PI restart? Why do you think that this issue makes Pi to restart?
What is the log?

Reply:
If just calling `config.sh` makes Pi restart, there is really a big bug in `config.sh` or somewhere else, or because the bad implementation of this whole access point server. Putting `&` is just to "hide" the bug and that is dangerous. It causes debugging more difficult in the future.

Anyway, that is just assumption, as long as I don't see the log of that "restart" issue.

I cann't approve this PR, as long as you still refuse to read the doc carefully and fix the code to follow the doc.

Reply:
When we run `config.sh` it, in turn,it runs the script `rwap.sh` which removes the Pi from AP mode which in turn requires a reboot. 
Also, please give the link of the doc that you are suggesting

Reply:
If we can't go with using '&' , I can create another endpoint which will just reboot the pi. And hence not running the other script in the background